.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.2.4
```

Comment:
```suggestion
2.2.4
```

Reply:
Also curious why you'd set the Ruby version that far back? 2.2 is EOL. Should be `2.5` or `2.6` at least.

Reply:
For people who aren't able to upgrade and are still wanting to use Upsert.  That being said, I suppose I'm not really planning on back porting new features anyway.